frontend/apps/erd-web/.env.production

```diff
@@ -0,0 +1 @@
+NEXT_PUBLIC_ENV_NAME=production
```

Comment:
Since `.env.*` is determined from NODE_ENV, so `.env.preview` cannot be provided.
Therefore, preview settings are set from Vercel.